handbook/product/README.md

```diff
@@ -81,13 +81,12 @@ changing specifications while ensuring that Fleet meets our brand and quality gu
 You'll know it's time for expedited drafting when:
 - The team discovers that a drafted user story is missing crucial information that prevents contributors from continuing the development task.
 - A user story is taking more effort than was originally estimated, and Product Manager wants to find ways to cut aspects of planned functionality in order to still ship the improvement in the currently scheduled release.
+- A user story on the drafting board won't reach "Settled" by the last estimation session in the current sprint.
```

Comment:
I realized we should revise this to indicate that missing the last estimation session doesn't automatically qualify for expedited drafting. By default, it would wait until the next sprint. There would need to be an urgent reason for it to be included in the current sprint.

Reply:
```suggestion
- A user story on the drafting board won't reach "Settled" by the last estimation session in the current sprint and cannot wait until the next sprint. 
```